README.md

```diff
@@ -0,0 +1,96 @@
+# Aspose.Imaging Cloud Python SDK 
```

Comment:
@s-zubov Please, enrich the README with info about async requests in a manner which is used in [PHP SDK](https://github.com/aspose-imaging-cloud/aspose-imaging-cloud-php/blob/master/README.md#asynchronous-api). Thanks.

Reply:
Thank you, I've provided information about async requests at the readme.